src/snapred/ui/view/BackendRequestView.py

```diff
@@ -28,5 +29,8 @@ def _labeledCheckBox(self, label):
     def _sampleDropDown(self, label, items=[]):
         return SampleDropDown(label, items, self)
 
+    def _multiSelectDropDown(self, label, items=[]):
```

Comment:
**A key question here:  do _all_ `BackendRequestView` actually _have_ a multi-select dropdown?**  If so, is it always the "pixel masks" dropdown?  If not, this possibly should be moved to `ReductionRequestView`.

Reply:
I left this integration here because all of our views are derived from the `BackendRequestView`. Previously, `ReducitonRequestView` was made into a QtWidget due to a the lack of the existence of `ReductionService` when `ReductionRequestView` was created. In order to merge, I needed a working model of this view and due to the lack of service backend calls to retrieve information, I couldn't follow the "traditional" SNAPRed pattern of implementing the view.

By 'traditional', I mean specifically this:
```python
class SomeView(BackendRequestView):
   def __init__(self, some arguments taken from the workflow (which are produced by a service call within workflow), parent=None):
# etc.... 
``` 

With the service created, I decided to revert this view to fit the 'standard' model views follow within SNAPRed and adding the functionality of the multi-select drop down GUI widget to `BackendRequestView` to expose this functionality to other potential views later in the future.